docs/user/openstack/deploy_on_ovs_dpdk.md

```diff
@@ -0,0 +1,94 @@
+When deploying vms on to this type of PaaS, the correct hugetable size and cpu pinnings need to be configured for the vms. 
+
+
+This maybe accomplished in two ways: 
+
+
+2) The flavors used for the OCP are defined with appropriate values. For example: 
+   ```
+   $opensack openstack flavor create dpdk_flavor --ram 16384 --disk 45 --vcpus 4
+   $ openstack flavor set \
+    --property dpdk=true \
+    --property hw:cpu_policy=dedicated \
+    --property hw:mem_page_size=large \
+    dpdk_flavor.small
+   ```
+
+3) The image properties maybe be defined in the install-config.yaml using [platform.openstack.
+   clusterOSImageProperties][customization.md]. For example:
+   ```
+   platform:
+      openstack:
+         clusterOSImageProperties:
+            - hw_cpu_policy: dedicated
+            - hw_mem_page_size: 1GB
+   ```
+   The flavor used to deploy the images must allow the image property values selected. For example, if you attempt 
+   to deploy a vm using an iamge with the property hw_mem_page_size: 1GB, but the flavor being used is set with 
+   --property hw:meme_page_size=small, the deployment of the vm will fail because the flavor does not allow for 
+   using a 1GB page size. The flavor property would need to either not be set, which is the same as 
+   hw:mem_page_size=any, or the flavor property should be set to hw:mem_page_size=large.
+
+
+## OpenShift cluster deployment:
+### Create aggregate and flavor
+Create a ‘dpdk’ aggregate group and flavor to use for the OpenShift cluster, [per the OSP documentation][1]
+First, [create the aggregate group:
+```
+$ openstack aggregate create dpdk_group
+$ openstack aggregate add host dpdk_group [compute-host]
+$ openstack aggregate set --property dpdk=true dpdk_group
+```
+
+Next, create a flavor with at least 16GB RAM, 45GB of disk and 4 CPUs, and then set  the relevant flavor 
+properties:
+```
+$ openstack flavor create dpdk_flavor --ram 16384 --disk 45 --vcpus 4
+$ openstack flavor set \
+    --property dpdk=true \
+    --property hw:cpu_policy=dedicated \
+    --property hw:mem_page_size=any \
+    --property hw:numa_nodes=1 \
+    dpdk_flavor.small
+```
+
+You may also wish to configure multiqueue for the guest image:
+```
+$ openstack image set \
+    --property hw_vif_multiqueue_enabled=true \
+    <core_os_image>
+```
+where core_os_image is the name of the image to be used for RHCOS. This will be the value you later enter for platform.openstack.clusterOSImage. This could also be accomplished by using platform.openstack.clusterOSImageProperties in install-config.yaml
+
+### Create low latency network if required.
+The low latency network is a network which will be used to create ports which will be used by the application 
+wanting to use the poll mode drivers to access the network. 
+```
+$ openstack network create low_latency
+$ openstack subnet create \
+    --network low_latency \
+    --subnet-range 192.168.200.0/24 \
+    low_latency_subnet
+```
```

Comment:
I asked this elsewhere, but aren't all networks "low latency" networks if OVS-DPDK is in use? What other networks would there be? Everything will be going through the same switch, right?